### DISCUSSION TITLE
Fejl: Upublicerede events er publicerede og åbne via arrangørside

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#27] [Bugs] [answered]
Selvom et event ikke er godkendt til udgivelse - hos en arrangør med manuel godkendelse - så kan man med det samme tilmelde sig eventet, hvis man går via arrangørsiden. Eventet ligger altså ikke på portalen, men ligger offentligt via arrangørsiden. 

Det er vigtigt, at et event er publiceret af både arrangør og frivilligorganisation på hele portalen, før det er tilgængeligt. 

Se f.eks. Stella Polaris og VM i Stand Up Paddeling og Copenhagen Paddle Festival hos Kulturens Frivillige.

**Comment by mikkelsl** [comment, marked as answer]
Det er et kendt problem: https://github.com/orgs/xooom-community/projects/2?pane=issue&itemId=60857778

Jeg flytter den til Development, så vi får den prioriteret.
